include/sdf/Model.hh

```diff
@@ -508,6 +508,14 @@ namespace sdf
     /// \param[in] _plugin Plugin to add.
     public: void AddPlugin(const Plugin &_plugin);
 
+    /// \brief Calculate and set the inertials for all the links belonging
+    /// to the model object
+    /// \param[in] _config Custom parser configuration
+    /// \param[out] _errrors A vector of Errors objects. Each errors contains an
```

Comment:
```suggestion
    /// \param[out] _errors A vector of Errors objects. Each errors contains an
    /// \param[in] _config Custom parser configuration
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)